packages/byods/jest.setup.js

```diff
@@ -0,0 +1,12 @@
+expect.extend({
+    toBeCalledOnceWith(received, ...expected) {
+      try {
+        expect(received).toBeCalledTimes(1);
+        expect(received).toBeCalledWith(...expected);
+      } catch (error) {
+        return {message: () => error, pass: false};
+      }
+  
+      return {pass: true};
+    },
```

Comment:
What does this file do?

Reply:
This I added to support custom jest matcher